packages/core/data-transfer/package.json

```diff
@@ -27,11 +27,13 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "scripts": {
-    "build": "tsc -p tsconfig.json",
     "clean": "rimraf ./dist",
+    "build": "yarn build:ts",
```

Comment:
@Convly Do we need this, since in the main package.json we're running build:ts and build as two steps? It seems like this will cause it to build twice.

Reply:
Nice catch indeed. I think we can remove it :slightly_smiling_face: